skyportal/tests/api/test_gcn.py

```diff
@@ -637,27 +637,31 @@ def test_gcn_summary_observations(
 
     # wait for the executed observations to populate
 
-    data = {
+    params = {
         'telescopeName': name,
         'instrumentName': instrument_name,
         'startDate': '2019-08-13 08:18:05',
         'endDate': '2019-08-19 08:18:05',
     }
     nretries = 0
     observations_loaded = False
-    while not observations_loaded and nretries < 26:
+    while not observations_loaded and nretries < 25:
         try:
             status, data = api(
-                'GET', 'observation', params=data, token=super_admin_token
+                'GET', 'observation', params=params, token=super_admin_token
             )
             assert status == 200
             data = data["data"]
-            assert len(data['observations']) == 10
+            assert len(data['observations']) >= 9
```

Comment:
Yes. First I noticed the test was wrong, I'm adding 9 obs not 10. Plus adding a superior instead of an equal fixes some issues with other tests also adding some obs.